yetanotherco/aligned_layer
# feat(telemetry): Add url data to operator updates on telemetry service

### Description

This PR adds new data to be updated for each operator on the telemetry service, which includes:
* `eth_rpc_url`
* `eth_rpc_url_fallback`
* `eth_ws_url`
* `eth_ws_url_fallback`
* `version` (this one was already present)

The implementation is also updated from the operator's side.

### Testing

In order to test, the following steps were taken:

1. Start anvil and the aggregator with:
 - `make anvil_start_with_block_time`
 - `make aggregator_start`
2. Build and register operator with:
 - `make build_operator`
 - `make operator_full_registration CONFIG_FILE=config-files/config-operator-1.yaml`
3. Start telemetry service with:
 - `make telemetry_clean_db`
 - `make telemetry_run_db`
 - `make telemetry_start`
 4. Go to `http://localhost:4001/api/operators` and you should see the operator with its data fields. Some of them, like the version and RPC related should be set as null, since the operator is not up yet.
 5. Start the operator with `./operator/build/aligned-operator start --config ./config-files/config-operator-1.yaml`
 6. Go to `http://localhost:4001/api/operators` and you should see the operator, now with all fields with a certain value, including version and RPC related.

It is also highly advisable to test previous functionality as it's described on [this already merged PR](https://github.com/yetanotherco/aligned_layer/pull/1077)
